src/common/decimal/src/decimal128.rs

```diff
@@ -96,10 +96,24 @@ impl Decimal128 {
         self.scale
     }
 
-    /// Convert to ScalarValue
+    /// Convert to ScalarValue(value,precision,scale)
     pub fn to_scalar_value(&self) -> (Option<i128>, u8, i8) {
         (Some(self.value), self.precision, self.scale)
     }
+
+    /// Convert to PbDecimal128, the PbDecimal128 is represented as (high-64 bit, low-64 bit),
+    /// and the precision and scale is not stored in PbDecimal128.
+    ///
+    /// Return (high-64 bit, low-64 bit)
+    pub fn to_pb_decimal128(&self) -> (i64, i64) {
+        ((self.value >> 64) as i64, self.value as i64)
+    }
+
+    /// Convert from PbDecimal128
+    pub fn from_pb_decimal128(hi: i64, lo: i64, precision: u8, scale: i8) -> Self {
```

Comment:
Now there is no `PbDecimal128`. We might consider other names for these methods.

e.g. The first one splits value into parts, maybe we can rename it to `split_value`? The second one is `from_value_precision_scale()`.